www/css/index.css

```diff
@@ -2,6 +2,7 @@ a:active {
   color:#d35400;
 }
 
+
```

Comment:
Remove empty line xD

Reply:
fuck you

Reply:
+1

Reply:
You are fired. Pack up your things. I don't want to see you here next Monday.